### PR TITLE
Add alternate links

### PR DESCRIPTION
**Pull request by kvdnberg**
- Add Alternate class and add array of Alternates to Url
- Add generation of xhtml alternate links in template
- Add test

### CODE REVIEW

File: Sitemap/Alternate.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+declare(strict_types=1);
```
Comment (thanosp) [inline Sitemap/Alternate.php]: You can't force php7 on the project

**Review by marina-endouble — APPROVED** [review]

File: Sitemap/Alternate.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+namespace Werkspot\Bundle\SitemapBundle\Sitemap;
+
+class Alternate
```
Comment (thanosp) [inline Sitemap/Alternate.php]: Maybe AlternateLink?
Also add a docblock

File: Sitemap/Alternate.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    /**
+     * @var string
+     */
+    private $hreflang;
+
+    /**
+     * @param string $href
+     * @param string $hreflang
+     */
+    public function __construct(string $href, string $hreflang)
```
Comment (thanosp) [inline Sitemap/Alternate.php]: Can't use string type

File: Tests/Controller/GenerateControllerTest.php [hunk 4]
```diff
@@ -95,6 +96,55 @@ public function testSectionAction()
         $this->assertGreaterThan(0, $client->getResponse()->getMaxAge());
     }
 
+    public function testSectionActionWithAlternates()
```
Comment (thanosp) [inline Tests/Controller/GenerateControllerTest.php]: Missing a test without alternates

**Review by thanosp — CHANGES_REQUESTED** [review]
Also missing docs

File: Sitemap/Alternate.php (hunk 3 above)
Comment (marina-endouble) [reply Sitemap/Alternate.php]: Why? :)

File: Tests/Controller/GenerateControllerTest.php (hunk 4 above)
Comment (kvdnberg) [reply Tests/Controller/GenerateControllerTest.php]: Added a line to the test above to check there are no xhtml tags there.

File: README.md [hunk 5]
```diff
@@ -146,3 +146,39 @@ class ProductSitemapProvider extends AbstractSitemapProvider
...
+        $urlRoute = $this->generateUrl('home');
+        $urlRouteDe = $this->generateUrl('home', ['_locale' => 'de']);
+        $urlRouteFr = $this->generateUrl('home', ['_locale' => 'fr']);
+        
+        $sitemapUrl = new Url($urlRoute, Url::CHANGEFREQ_WEEKLY, 1.0);
+        $sitemapUrl->addAlternateLink(new AlternateLink($urlRouteDe, 'de'));
+        $sitemapUrl->addAlternateLink(new AlternateLink($urlRouteFr, 'fr'));
+        $sitemapUrl->addAlternateLink(new AlternateLink($urlRoute', 'x-default')); // Country select page
+        // Or
+        $sitemapUrl->addAlternateLink(new AlternateLink($urlRoute', 'en));
```
Comment (thanosp) [inline README.md]: Quote issue

**Review by thanosp — CHANGES_REQUESTED** [review]

**Review by thanosp — APPROVED** [review]

**Pull request merged**